### ISSUE TITLE
[Dropdown] - search selection has 1st item already selected

### ISSUE THREAD

**Issue by avalanche1** [Semantic-Org/Semantic-UI#3813]
When opening search selection dropdown - it has 1st item already selected. Which contradicts to the UX of other dropdown types.

**Comment by Thomas12** [comment]
Has this been resolved or is there a workaround?

**Comment by avalanche1** [comment]
With 855 open issues this particular one might just not be a priority..

**Comment by Thomas12** [comment]
At least for my scenario there is a workaround, see bug https://github.com/Semantic-Org/Semantic-UI/issues/4709.

**Comment by avalanche1** [comment]
Well, setting `forceSelection: false` doesn't help in my case:
https://jsfiddle.net/avalanche1/0sruz8sh/ - Angular is selected by default when opening dropdown

**Comment by xxRockOnxx** [comment]
weird, mine doesn't do what you said. I'm on Chrome 54 (Linux)

**Comment by saitonakamura** [comment]
You are talking about different 'selection'
The one @Thomas12 is talking about is semantic ui dropdown `forceSelection` option which selects the first element (matching search criteria if that's search dropdown) even when you close it without actually selecting an item
The second one, @avalanche1 is talking about is that you see the highlight of the first element

**Comment by xxRockOnxx** [comment]
@saitonakamura I was about to say that. I was about to reply if he meant the selected or highlighted

**Comment by avalanche1** [comment]
@saitonakamura , thanks for clarifying! I've edited topic accordingly

**Comment by xmoroccan** [comment]
Yeah, so an others unwanted behavior, when i use `ui.dropdown.multiple.search` with `allowAdditions = true`, in the results i have something like `tester` as a value, and when i type for example: `test` and click on enter or type a delimiter such a `,` drowpdown automatically chose the first highlighted  value, even if use set `forceSelection = false`
any suggestions !

**Comment by imrimt** [comment]
@xmoroccan I'm having the same issue as well, and it's pretty much impossible to add a new value if that value is a substring of another already existing value. I've tried both `forceSelection = false` and `selectOnKeydown = false` (to at least enable adding new value on delimiter) but none of this has worked. 

Would love to have a solution/workaround for this as it has been bugging me for ages.

**Comment by MarcusEFC** [comment]
Has anyone found a solution for this?
